### DISCUSSION TITLE
Backslash and import from CSV

### DISCUSSION THREAD

**Discussion by sryze** [dbeaver/dbeaver#19315] [Q&A]
I've exported a table through DBeaver to a CSV file, but when I imported it, the backslashes were surprisingly removed, supposedly because they were treated as an escape character. For example, `\d` was replaced with just `d` (inside a regex string). I left the default settings unchanged.

So I have two questions:

1. What's the setting to escape `\` during export? There is none in this dialog:
   ![image](https://user-images.githubusercontent.com/4525736/224229632-00d89132-6169-49d3-a4c0-fa248379b24f.png)
2. Even if I set "Escape Character" to an empty string during import, `\` is still removed from strings afterwards. Does this setting default to `\` if it's empty?
   ![image](https://user-images.githubusercontent.com/4525736/224229726-c5258fb4-284c-4bbb-9656-6f12023a198a.png)

**Comment by jpivac** [comment]
I encountered the same issue. I set the Escape char to something else (eg #) and it imported my \'s correctly.

**Reply by sryze** [reply]
That's a good workaround, thanks.

**Reply by piomar123** [reply]
Escape char can be a string, so using a long string unlikely to show up in the data (e.g. `~!@#$%^&*()`) is a safer option than a single character.
